ledger/src/shred.rs

```diff
@@ -80,7 +80,7 @@ mod common;
 mod legacy;
 mod merkle;
 mod shred_code;
-mod shred_data;
+pub mod shred_data;
```

Comment:
@behzadnouri do you have a preference in how the shred payload `capacity()` is exposed rather than making the module public?

Reply:
this is fine

Reply:
On a second thought, I think might be better to keep the module private, and instead move `ShredData` here:
https://github.com/solana-labs/solana/blob/deb13abb1/ledger/src/shred.rs#L54
 to `pub use` just below it:
```diff
diff --git a/ledger/src/shred.rs b/ledger/src/shred.rs
index 4c20924c96..01bfefd344 100644
--- a/ledger/src/shred.rs
+++ b/ledger/src/shred.rs
@@ -51,9 +51,8 @@
 
 #[cfg(test)]
 pub(crate) use shred_code::MAX_CODE_SHREDS_PER_SLOT;
-pub(crate) use shred_data::ShredData;
 pub use {
+    self::shred_data::ShredData,
     self::stats::{ProcessShredsStats, ShredFetchStats},
     crate::shredder::Shredder,
 };
 use {
```

Reply:
hmm, this doesn't mark the module `shred_data` public. leaving this as-is.

Reply:
done